_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+
+This link, with a background with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: black;
+		background-color: #indianred;
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 4
+
+This element with a [semantic role][] that inherits from link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: underline;
+	}
+</style>
+<p>This is indicated in the cost of a <a href="#bcc0f155" role="doc-glossref">credit default swap</a>.</p>
+<h1>Glossary</h1>
+<p id="bcc0f155">
+	A credit default swap (CDS) is a financial derivative or contract that allows an investor to "swap" or offset his or
+	her credit risk with that of another investor.
+</p>
+```
+
+#### Passed Example 5
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable color because the color contrast is 4.98.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: crimson;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a class="test" href="http://w3.org/WAI">WAI webpage</a>.
+</p>
+```
+
+#### Passed Example 6
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable bottom border when it is hovered or focused.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	a:hover,
+	a:focus {
+		border-style: solid;
+		border-color: red;
+		border-width: 0px;
+		border-bottom-width: 1px;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 7
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable box-shadow.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+		box-shadow: 4px 4px;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 8
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable font-style when it is focused.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	a:focus {
+		font-style: italic;
```

Comment:
This shouldn't be what makes this pass. It needs at least a hover style too.

Reply:
Current expectation is that if there is a way to distinguish the link from the other line text either on hover, on focus, or with none of them, then the rule passes. It doesn't require it to be both on focus and hover. There is nothing in the SC stating that it needs to happen on both. So we might be passing the rule when the SC is not satisfied, but certainly enforcing the fail-on-fail relationship.

Reply:
Just to clarify something: passing this rule does not mean SC 1.4.1 is satisfied. However, failing the rule means SC 1.4.1 is not satisfied. 

So, what we are saying in the rule is that, if the link does not have something that distinguishes it from the surrounding text in one of 3 scenarios (always, when hovered, or when focused) then it fails SC 1.4.1. When it does have something to distinguish it in at least one of those scenarios we cannot fail it. Given that SC 1.4.1 does not require the distinguishing style to exist both on hover and focus we cannot require that to fail.

Reply:
Examples updated so that all examples which are not meeting the new expectation 3 were moved to failed examples.